package.json

```diff
@@ -16,7 +16,7 @@
     "node-gyp-build": "^4.8.0"
   },
   "peerDependencies": {
-    "tree-sitter": "^0.21.1"
+    "tree-sitter": "^0.22.1"
```

Comment:
Always confuses me that this version doesn't match `tree-sitter --version` :sweat_smile:

Reply:
It will only match if you install treesitter with npm.